core/src/main/resources/error/README.md

```diff
@@ -24,27 +24,27 @@ Throw with arbitrary error message:
 
 ### After
 
-`error-class.json`
+`error-classes.json`
 
     "PROBLEM_BECAUSE": {
-      "message": ["Problem %s because %s"],
+      "message": ["Problem <problem> because <cause>"],
       "sqlState": "XXXXX"
     }
 
 `SparkException.scala`
 
     class SparkTestException(
         errorClass: String,
-        messageParameters: Seq[String])
+        messageParameters: Map[String, String])
       extends TestException(SparkThrowableHelper.getMessage(errorClass, messageParameters))
         with SparkThrowable {
         
-      def getErrorClass: String = errorClass
+      override def getErrorClass: String = errorClass
```

Comment:
Need to override `getMessageParameters` (and return `messageParameters.asJava`) too otherwise the example won't work.

Reply:
Updated the example. Thanks!